cypress/e2e/3-testing/login.cy.js

```diff
@@ -0,0 +1,70 @@
+// login_spec.js
+
+describe('Login Functionality', () => {
+  beforeEach(() => {
+    cy.visit('127.0.0.1:5500');

```

Comment:
```suggestion
    cy.visit('/');
```

This will hit Cypress' test server by default